iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -2760,6 +2765,12 @@ +(void)setupUNUserNotificationCenterDelegate {
     [OneSignalHelper registerAsUNNotificationCenterDelegate];
 }
 
++(BOOL) shouldDisableBasedOnProcessArguments {
+    if ([NSProcessInfo.processInfo.arguments containsObject: @"DISABLE_ONESIGNAL_SWIZZLING"]) {
```

Comment:
Nit comment, is ok to have the scape between `containsObject: @"DISABLE_ONESIGNAL_SWIZZLING"` ?